app/services/flow/flows/provider_merits.rb

```diff
@@ -2,8 +2,8 @@ module Flow
   module Flows
     class ProviderMerits < FlowSteps
       STEPS = {
-        start_merits_assessments: {
```

Comment:
Renaming this step will cause problems for any in-progress applications that have a `provider_step` of `start_merits_assessments`. If a provider tries to progress them after this has been released it will error (I know this because it happened after I renamed the `respondent` step to `opponent` 😬). There are currently 16 applications in production at the `start_merits_assessments` step.

I think we need a small rake file that finds all the applications at that step and updates them to the new step. We can run it manually after your branch has been deployed. There are some examples of similar tasks in `lib/tasks/`.

I think this one would need to do something like
```
LegalAidApplication.where(provider_step: 'start_merits_assessments').each do |app|
  app.provider_step = 'start_chances_of_successes'
  app.save
end
```

possibly with a rollback method too, in case we need to revert it for some reason.

Reply:
@mpw5 this makes sense, hadn't thought of that at all. Thanks for the info too. I'm not sure I know how to create a rollback task. I'll get one of the devs not on leave to pair on that!

Reply:
@eloisetait3 I think it would be similar to the `migrate` task above, just going in the opposite direction, eg:

```
task rollback: :environment do
  LegalAidApplication.where(provider_step: 'start_chances_of_successes').each do |app|
    app.provider_step = 'start_merits_assessments'
    app.save
  end
end
```

Sorry for requesting changes and then disappearing off on leave!